### PR TITLE
Add task status component

### PR DESCRIPTION
**Pull request by vitalii-bulyzhyn**
PR contains:
- displaying task status on the annotation UI

### CODE REVIEW

**Comment by vitalii-bulyzhyn** [comment]
@PJEstrada can you please take a look on the branch?

File: frontend/src/components/annotation/task_status.vue [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import Vue from "vue";
+
+  export default Vue.extend( {
+    name: 'task_status',
+    props: {
+        task_status: {
+            type: Boolean,
+            required: true
+        },
+        allo_reviews: {
```
Comment (PJEstrada) [inline frontend/src/components/annotation/task_status.vue:48]: Small typo here on props

**Review by PJEstrada — CHANGES_REQUESTED** [review]
Hey Vitalii thanks for putting this together.

I have tested the branch, but I'm getting errors on the chrome console and the task completion button does not seem to update the status correctly on the frontend (although the request does gets send to the server and response is 200)

image


Can you please check this one out ?

**Comment by vitalii-bulyzhyn** [comment]
@PJEstrada thanks for pointing those things out, just fixed them

**Comment by vitalii-bulyzhyn** [comment]
@PJEstrada everything seems to work fine now

**Review by PJEstrada — APPROVED** [review]
Thanks Vitalii ! Code looks good now I think we can merge now to master :)

**Pull request merged**